onnxruntime/contrib_ops/cpu/tokenizer.cc

```diff
@@ -645,10 +497,11 @@ Status Tokenizer::Compute(OpKernelContext* ctx) const {
   if (char_tokenezation_) {
     s = CharTokenize(ctx, N, C, input_dims);
   } else {
-    if (regex_ != nullptr) {
-      s = ExpressionTokenize(ctx, N, C, input_dims);
-    } else {
+    if (!separators_.empty()) {
       s = SeparatorTokenize(ctx, N, C, input_dims);
+    } else {
+      assert(regex_ != nullptr);
+      s = ExpressionTokenize(ctx, N, C, input_dims);
```

Comment:
I guess this `Expression` means `TokenExpression` while `SeparatorTokenizer` will become `SeparatorExpressionTokenizer`.

Reply:
Sorry for making confusion. Could we rename TokenExpression to TokenExpressionTokenize and SeparatorExpressionTokenizer back to SeparatorExpressionTokenize?


---
In reply to: [276864043](https://github.com/Microsoft/onnxruntime/pull/857#discussion_r276864043) [](ancestors = 276864043)